.storybook/preview.tsx

```diff
@@ -15,10 +18,24 @@ export const parameters = {
   },
 };
 
+const globalStyles = css`
+  ${defaultFontFiles({ baseUrl: "/" })}
```

Comment:
I'm curious about the baseUrl here. I think the default font file assets are supposed to be loaded from a Metabase instance. How are we able to get the font files from within Storybook, as it has no link to a Metabase instance?

Reply:
@deniskaber oh nice, I was wondering what that line does. Thanks!